docker/build-tools/Dockerfile

```diff
@@ -277,16 +292,18 @@ RUN rm -fr /usr/local/lib/python3.5/dist-packages/pip
 
 FROM ubuntu:xenial as base_os_context
 
-# required for golang: ca-certificates, gcc, libc-dev, git
+# required for binary tools: ca-certificates, gcc, libc-dev, git, iptables, libltdl7
```

Comment:
it would be helpful to document each install dependency and why it is required - for future distro changes that may rename packages.

Reply:
I do, that's why I added the entries I did. The dependencies are per context.